cmake/RPackageInstall.cmake.in

```diff
@@ -27,7 +27,7 @@ file(WRITE "${build_dir}/R-package/src/Makevars.win" "all:")
 
 # Install dependencies
 set(XGB_DEPS_SCRIPT
-    "deps = setdiff(c('data.table', 'magrittr', 'stringi'), rownames(installed.packages())); if(length(deps)>0) install.packages(deps, repo = 'https://cloud.r-project.org/')")
+    "deps = setdiff(c('data.table', 'magrittr'), rownames(installed.packages())); if(length(deps)>0) install.packages(deps, repo = 'https://cloud.r-project.org/')")
```

Comment:
Good catch! We should also add `jsonlite` to this list, since it's now a dependency of XGBoost.

Reply:
oh sure! Would you like me to add `{jsonlite}` and `{Matrix}` (the other `Imports`-level, non-default dependencies from https://github.com/dmlc/xgboost/blob/8760ec48277b345aaaa895b82570c25566fc0503/R-package/DESCRIPTION#L61) in this PR?

Reply:
I thought `Matrix` was part of the base R?

Reply:
Never mind, yes, we should add it.

Reply:
ok, will add it! For reference in the future, you can see the list of defaults at https://cran.r-project.org/doc/manuals/R-admin.html#Default-packages

Reply:
added in https://github.com/dmlc/xgboost/pull/6906/commits/bcddaadd050a544e541647dad2d339211324b6a9